octobot_trading/constants.py

```diff
@@ -136,7 +137,19 @@
 # text content of errors due to exchange compliancy rules
 EXCHANGE_COMPLIANCY_ERRORS: typing.List[typing.Iterable[str]] = [
     # OKX ex: Trading of this pair or contract is restricted due to local compliance requirements
-    ("restricted ", "compliance"),
+    ("restricted", "compliance"),
+]
+
+# text content of errors due to exchange compliancy rules
+EXCHANGE_ORDER_NOT_FOUND_ERROR: typing.List[typing.Iterable[str]] = [
+    # coinbase {"error":"NOT_FOUND","error_details":"order with this orderID was not found",
+    #   "message":"order with this orderID was not found"}
+    ("not_found", "order"),
```

Comment:
Couldn't we specify it in the exchange tentacle?

Reply:
indeed, it's up